release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md

```diff
@@ -743,6 +725,71 @@ To view bug fixes, see the GitHub milestone for 2201.2.0 (Swan Lake) of the repo
 - [Language Server](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed+label%3ATeam%2FLanguageServer)
 - [update tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed+label%3AType%2FBug)
 
+##### JSON to record converter
+Improved JSON value to Ballerina record conversion logic.
+
+###### Handling of array of elements/objects
+Instead of looking at the first object of the array, all elements/objects would be looked at to generate the record field.
+```json
+[
+    { "id": "5001", "type": "None", "index": "15" },
+    { "id": "5002", "type": "Glazed", "index": 15 }
```

Comment:
```suggestion
    { "id": "5001", "index": "15" },
    { "id": "5002", "index": 15 }
```

Reply:
@AzeemMuzammil Shall we simplify the complexity of the other example JSONs also, to emphasise the improvement point more clearly?

Reply:
Sure